rrfs-test/ush/envpy

```diff
@@ -0,0 +1,4 @@
+#!/bin/tcsh
+setenv PYTHONPATH "/home/Donald.E.Lippi/miniconda3:/home/Donald.E.Lippi/Python/conda/local/emcpy/develop/lib/python3.9/site-packages:/home/Donald.E.Lippi/Python/conda/local/PyGSI/develop/lib/python3.9/site-packages:/home/Donald.E.Lippi/bin/python:$PATH"
```

Comment:
Does one of  RDASapp two env  setup  (RDAS and EVA) be able to provide conda env needed here? 
if it is possible,  I suggest to use  EVA in RDASapp to provide required python (and all visualization/postprocessing related packages)

Reply:
I agree with Ting, whatever you need, we should add to a more standardized environment

Reply:
It seems that the only thing the currently loaded python environment doesn't have is matplotlib and cartopy. I also have a custom colormap.py script that handles near zero well for difference maps that it would be nice to also make available. Do either of you have suggestions on how to add these? I will go ahead and remove this file from the PR.

Reply:
the EVA env has both of those. The colormap script we can include in the repo perhaps?

Reply:
Ok thanks for that info. I just tested via loading the EVA env. It works. I just added my colormap.py script. Not sure if the ush was the best place.